project4/3700crawler

```diff
@@ -15,288 +22,413 @@ CRLF = '\r\n'
 # Maximum amount of data to be received at once from the socket
 MAX_BYTES = 5000
 
+
 # ------------------------- END OF GLOBALS VARIABLES -------------------------
 
 
 # -------------------------- HTTP UTILITY FUNCTIONS --------------------------
```

Comment:
We can probably remove the "HTTP UTILITY FUNCTIONS" comment separators since they aren't really utility functions anymore (unless you want to rename "functions" to "classes").